firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/TestUtils.java

```diff
@@ -22,12 +22,14 @@
 final class TestUtils {
   private TestUtils() {}
 
-  static void assertTaskFailure(Task task, Status status, String messageSubstring) {
+  static FirebaseAppDistributionException assertTaskFailure(
+      Task task, Status status, String messageSubstring) {
     assertThat(task.isSuccessful()).isFalse();
     assertThat(task.getException()).isInstanceOf(FirebaseAppDistributionException.class);
     FirebaseAppDistributionException e = (FirebaseAppDistributionException) task.getException();
     assertThat(e.getErrorCode()).isEqualTo(status);
     assertThat(e).hasMessageThat().contains(messageSubstring);
+    return e;
```

Comment:
Why now return a firebase exception?

Reply:
So I can use the returned exception to make assertions about the release (like in `updateAppTask_onAppResume_setsUpdateCancelled`)

Reply:
I think this is fine to keep as is. I am not seeing the impact of asserting on the release im aware that was in place before you touched the test, just not sure what the intent with that is. Either way looks good to me.

Reply:
OK sounds good. I do think there's benefit to asserting that the release was set on the exception. I'll leave it as is for now.